_layouts/about.html

```diff
@@ -5,17 +5,9 @@
 <div class="post">
 
   <header class="post-header">
-    {% assign name = site.title | split: ' ' %}
-    {% assign firstname = name | first %}
-    {% if name.size > 2 %}
-       {% assign middlenames =  name | slice:1, name.size-1 | join:" " %}
-    {% else %}
-       {% assign middlenames = '' %}
-    {% endif %}
-    {% assign lastname = name | last %}
-    <a class="navbar-brand title font-weight-lighter" href="{{ site.baseurl | prepend: site.url }}/">
-     <span class="font-weight-bold">{{ firstname }}</span> {{ middlenames }}  {{ lastname }}
-    </a>
+    <h1 class="post-title">
+     <span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }}  {{ site.last_name }}
```

Comment:
```suggestion
     {% if site.title %}{{ site.title }}{% else %}<span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }}  {{ site.last_name }}{% endif %}
```
Same here, could you apply the above change?